test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -315,4 +846,435 @@ test.describe( 'Links', () => {
 			},
 		] );
 	} );
+
+	test.describe( 'Editing link text', () => {
+		test( 'should not display text input when initially creating the link', async ( {
+			page,
+			editor,
+			pageUtils,
+		} ) => {
+			// Create a block with some text.
+			await editor.insertBlock( {
+				name: 'core/paragraph',
+			} );
+			await page.keyboard.type( 'This is Gutenberg: ' );
+
+			// Press Cmd+K to insert a link.
+			await pageUtils.pressKeys( 'primary+k' );
+
+			//Check that the HTML anchor input under Advanced is empty
+			await page.getByRole( 'button', { name: 'Advanced' } ).click();
+			const inputElement = page.getByLabel( 'HTML anchor' );
+			await expect( inputElement ).toHaveValue( '' );
```

Comment:
I do not understand this test at all. I'll need to run it and investigate what it's trying to assert.

Reply:
Yeh...it was testing totally the wrong thing. I fixed that so now it's asserting on the presence of the `Text` input within the Link Control as stated in the test description.